docker/Dockerfile.hdfs

```diff
@@ -76,10 +78,19 @@ ENV PATH $PATH:$HADOOP_HOME/bin:$HADOOP_HOME/sbin
 ARG JAVA_HOME_PATH
 ENV JAVA_HOME ${JAVA_HOME_PATH}
 ENV LD_LIBRARY_PATH ${JAVA_HOME_PATH}/lib/server:${LD_LIBRARY_PATH}
+ENV HADOOP_CONF_DIR /opt/hadoop/etc/hadoop/
+
+# set classpath
+RUN wget https://rw-yufan.s3.ap-southeast-1.amazonaws.com/hadoop_classpath_273.txt -P /root/
+
+# Need to copy the content in the hadoop_classpath_273.txt file, and then set env. 
+# Since the env command can only be written but not read, The following line is deleted here, and then added manually when packaging.
+
+# ENV CLASSPATH=/root/hadoop_classpath_273.txt:$CLASSPATH
```

Comment:
So we must set this file name into the classpath? Will a folder name satisfied this request?